.eslintrc.cjs

```diff
@@ -0,0 +1,84 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    'vitest-globals/env': true,
+  },
+  extends: [
+    'standard-with-typescript',
+    'plugin:react/recommended',
+    'plugin:prettier/recommended',
+    'plugin:@vitest/legacy-recommended',
+    'plugin:vitest-globals/recommended',
+    'plugin:@typescript-eslint/stylistic',
+
+    // 'plugin:vitest/recommended',
+  ],
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  overrides: [
+    {
+      env: {
+        node: true,
+      },
+      files: ['.eslintrc.{js,cjs}'],
+      parserOptions: {
+        sourceType: 'script',
+        include: ['src/**/*'],
```

Comment:
This is a bit confusing as overrides/files says the override is only related to `.eslintrc.(js|cjs)` and then this is saying it's got to do with `src/**/*`. If you meant for the override to only affect eslint's configuration file wouldn't it make sense for this line to be removed?